src/Components/Navbar/nav.jsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
 
+import { NavLink } from 'react-router-dom';
```

Comment:
This import is grouped with "route" which is your local logic.
Consider grouping this import with the rest of the third-party imports.

Reply:
I grouped them